roles/rest-api/handlers/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: start rest-api.service
+  service:
+    name: rest-api.service
+    state: started
```

Comment:
I believe you mean `restarted` here, don't you?

Reply:
yes